packages/yoga/src/Input/web/Password.jsx

```diff
@@ -120,6 +127,7 @@ const Password = ({ disabled, style, className, full, ...props }) => {
         onKeyDown={togglePassword}
         disabled={disabled}
         role="button"
+        aria-label={iconPasswordAriaLabel}
```

Comment:
I believe this button should behave like a checkbox, so we could just use the translation "Hide password" and the assistive technology would inform whether it is checked or not.

Another option is to have a prop for the "Hide password" label and another for "Show password" label, in this way, Yoga continues to define the text that is visible.

In the scenario of using only one translation for this button, we can inform whether it is active or not through `aria-pressed`.

Reply:
I created two props to receive the accessibility text. What do you think?

 @ https://github.com/gympass/yoga/commit/797178016f71733eab6499b0dbeb271e52514105